### ISSUE TITLE
Remove play-services

### ISSUE THREAD

**Issue by krt16s** [arpitkh96/AmazeFileManager#118]
As discussed on IRC, fdroid cannot build apps with play-services. Providing a  gradle build flavor will most likely work if you don't overly use play-services. Have a look at http://tools.android.com/tech-docs/new-build-system/user-guide#TOC-Product-flavors as a guide. Also look at the other app you mentioned on how to integrate them.

From a quick glance I see you update your build.gradle with:

````
    productFlavors {
        fdroid {
        }
        play {
        }
    }
````

Make non-free dependencies a `playCompile` instead of `compile` and add source sets, basically moving all code for non-free stuff to a specific source directory. E.g. QuickLyrics uses `QuickLyric/src/play/java/com/`. 

As I said, I am not an Android developer, but this should guide you in the right direction.

**Comment by vishal0071** [comment]
OK, I'll look into it. Thanks for guiding.

**Comment by vishal0071** [comment]
I've added a product flavor with name fdroid. It is not using the play services api and runs fine.
What to do next?

**Comment by krt16s** [comment]
Release another version and ping me :)

**Comment by vishal0071** [comment]
It's gonna take a while before we finalize new version. I'll inform you here once done.

**Comment by vishal0071** [comment]
Sir, we've released new version, so you might wanna take a look for fdroid variant (:

**Comment by vishal0071** [comment]
@krt16s could'nt find another way to communicate to you. 
I published a new version of Amaze here on GitHub. Last one was around a year ago and this one has a year worth changes. It'd be great if you could add a new build on the Fdroid application page.

**Comment by krt16s** [comment]
Well, actually you app was set to do automatic upgrades, however this only takes places once a day. Even if 24hrs had already passed, the update would have failed:

* The tag does not match the versionname ("v3.0.2 Beta 6" vs. " v3.0.2-beta.6")
* You included quite some jar files. Jar files are binaries that have to be replaced with source build versions. We allowed jcifs because we were less strict back then, it's a hassle to build and we verified it against upstream sources.

While I can try building them from source, I'd rather see that be done on your side. Both because I am rather short on time and because managing dependencies on our side is simply wrong :). Are those jars not available from central, jcenter or jitpack?
